darshan-util/pydarshan/benchmarks/benchmarks/report.py

```diff
@@ -61,3 +61,29 @@ def setup(self, dtype, darshan_logfile, mod, read_all):
     def time_mod_read_all_dxt_records(self, dtype, darshan_logfile, mod, read_all):
         self.report.mod_read_all_dxt_records(mod=mod,
                                              dtype=dtype)
+
+class ReportRecordsToDF:
+
+    params = [["examples/example-logs/ior_hdf5_example.darshan",
+               "examples/example-logs/dxt.darshan",
+               "tests/input/sample-dxt-simple.darshan",
+              ],
+              ["POSIX",
+               "LUSTRE",
+               "DXT_POSIX",
```

Comment:
I've added all of the modules in `ior_hdf5_example.darshan` since it covers all of the usual suspects, plus a few others. The list of modules + record counts for `ior_hdf5_example.darshan`:
```
Loaded Records: {'POSIX': 1, 'MPI-IO': 1, 'H5F': 3, 'H5D': 3, 'STDIO': 1, 'DXT_POSIX': 4, 'DXT_MPIIO': 4, 'LUSTRE': 1}
```
It has limited records for each module so other logs might be better suited to measuring performance differences. For example, `sample-badost.darshan` has 2048 `POSIX` and `LUSTRE` records.